### DISCUSSION TITLE
Predefined `readonly` Group

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#973] [Ideas]
Dear Developers,

I would like to propose to add a `readonly` group as predefined group to the `config/server-groups.json` (see: `package/src/main/config/server-groups.json`):
```
        "readonly": {
          "access": [],
          "resultSetLimit": -1,
          "readTimeout": -1,
          "types": {
            "*": {
              "access": [
                "readRecord"
              ]
            }
          }
        },
```
This would allow to create "safe" readonly users on-the-fly via the setting `server.defaultDatabases`.

**Comment by lvca** [comment]
I like it. @gramian would you like to work on this?

**Reply by gramian** [reply]
Cool. I will send a PR ASAP.
